cmd/start/main.go

```diff
@@ -53,7 +55,18 @@ func start(c *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	setupLog.Info("starting manager")
```

Comment:
why did we remove this log line?

Reply:
If you check the diff, it was replaced by:
```go
	setupLog.Info("Starting Tempo Operator",
		"tempo-operator", v.OperatorVersion,
		"default-tempo-image", v.DefaultTempoImage,
		"default-tempo-query-image", v.DefaultTempoQueryImage,
		"build-date", v.BuildDate,
		"git-hash", v.GitHash,
		"go-version", v.Go,
		"go-arch", runtime.GOARCH,
		"go-os", runtime.GOOS,
	)
```